storage/entities/account.js

```diff
@@ -322,6 +323,24 @@ class Account extends BaseEntity {
 		return this._getResults(filters, options, tx);
 	}
 
+	/**
+	 * Count total entries based on filters
+	 *
+	 * @param {filters.Account|filters.Account[]} [filters = {}]
+	 * @return {Promise.<Integer, NonSupportedFilterTypeError>}
+	 */
+	count(filters = {}) {
+		this.validateFilters(filters);
+
+		const mergedFilters = this.mergeFilters(filters);
+		const parsedFilters = this.parseFilters(mergedFilters);
+		const expectedResultCount = 1;
+
+		return this.adapter
+			.executeFile(this.SQLs.count, { parsedFilters }, { expectedResultCount })
+			.then(result => +result.count);
```

Comment:
There is a way to do same in Pg-Promise, but to make the implementation generic we can go with this approach.